integration-tests/subscriptions.test.ts

```diff
@@ -39,4 +46,39 @@ if (SUBSCRIPTIONS_ENABLED === 'true') {
             console.log(x);
         });
     });
+
+    let client: AxiosInstance;
+    describe('test subscription creation and deletion', () => {
+        beforeAll(async () => {
+            client = await getFhirClient({ tenant: MULTI_TENANCY_ENABLED ? 'tenant1' : undefined });
```

Comment:
`getFhirClient` already has the logic to default to tenant1 when multi-tennacy is enabled. There's no need to repeat that here.

In general most tests simply call `getFhirClient()` unless they need a specific role or a specific tenant( for tests that involve many tenants or for tenant isolation tests)

https://github.com/awslabs/fhir-works-on-aws-deployment/blob/401e97d9788a8998fbea707b80d5a0dd1aede87c/integration-tests/utils.ts#L79

https://github.com/awslabs/fhir-works-on-aws-deployment/blob/401e97d9788a8998fbea707b80d5a0dd1aede87c/integration-tests/utils.ts#L110